pkg/api/types.go

```diff
@@ -1146,10 +1146,97 @@ const (
 )
 
 // Affinity is a group of affinity scheduling rules, currently
-// only node affinity, but in the future also inter-pod affinity.
+// includes node affinity,pod affinity & pod anti affinity.
 type Affinity struct {
 	// Describes node affinity scheduling rules for the pod.
 	NodeAffinity *NodeAffinity `json:"nodeAffinity,omitempty"`
+	// Describes pod affinity scheduling rules for the pod.
+	PodAffinity *PodAffinity `json:"podAffinity,omitempty"`
+	// Describes pod anti affinity scheduling rules for the pod.
```

Comment:
Maybe "Describes pod anti-affinity scheduling rules (e.g. avoid putting this pod in the same node, zone, etc. as some other pod(s))."